tools/integration_tests/implicitdir/implicitdir_test.go

```diff
@@ -147,6 +149,35 @@ func createTempFile() string {
 	return fileName
 }
 
+// executing test
+func executeTest(flags []string, m *testing.M) (successCode int) {
+	for i := 0; i < len(flags); i++ {
+		if err := mountGcsfuse(flags[i]); err != nil {
+			log.Printf("mountGcsfuse: %v\n", err)
+			os.Exit(1)
```

Comment:
why not return the error directly, instead of convertinng that to successcode?

Reply:
fixed